packages/gatsby/src/utils/show-experiment-notice.ts

```diff
@@ -35,3 +45,24 @@ export function showExperimentNoticeAfterTimeout(
     clearTimeout(noticeTimeout)
   }
 }
+
+emitter.on(`COMPILATION_DONE`, () => {
+  emitter.off(`COMPILATION_DONE`, () => {})
```

Comment:
Does this work? Wouldn't `.off` require the same callback or some kind of token to disable that callback? Surely a fresh arrow isn't the way?

Doesn't `emitter` have a `.once` or something like that?

Reply:
> Doesn't emitter have a .once or something like that?
@pvdz You should know, you wrote it 😂 It doesn't, but the simple fix is to define the handler function first, then refer to it in both `on` and `off`. `once` would be a nice addition to mett though.

Reply:
Oh mett. Been a while. I just optimized it :p

Anyways, all the more reason this should be tested because this was a bug.

Reply:
oh right... I never find this behavior in event emitter intuitive for some reason. Thanks! And fixing.

Reply:
👍  to `once`